src/main/resources/helpTemplates/common.html

```diff
@@ -0,0 +1,53 @@
+<#--
+        This file contains part of the theming used to present Barclay docs on a website. Styling is separated
+        out, so pages will be minimalistic html unless replacement styling is provided.
+        -->
+
+    <#macro footerInfo>
+        <hr>
+        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
```

Comment:
If/when I find some time I'll make a PR to make the default output more generic, but I agree it's not a problem to keep this stuff in for now.